dev/cognitiveservices/data-plane/Language/examples/text/SuccessfulCustomHealthcarePostRequest.json

```diff
@@ -1,9 +1,8 @@
 {
   "parameters": {
     "Ocp-Apim-Subscription-Key": "{API key}",
-    "api-version": "2023-04-01",
+    "api-version": "2023-04-15-preview",
     "Endpoint": "{Endpoint}",
-    "jobId": "{Job ID}",
```

Comment:
Should this be kept?

Reply:
JobId is not passed in the POST request. It was there by mistake